### ISSUE TITLE
Support non "variant" GT-WT-01 sensors

### ISSUE THREAD

**Issue by martinhoefling** [jckuester/weather-station#13]
I would like to support the non "variant" version of the above sensors. They are defined in weather12.coffee and only differ by the binary ranges mapped to the values. How can we make a distinction here?
One idea would be to add a validation function to the Protocol to skip, one of the GT-WT-01 if the values are out of range. SeqLength etc. are all the same unfortunately.
Currently, I've just overwritten the ranges locally. @jckuester Let me know if you consider this a valid approach or if you have other ideas, I'll prepare a pull request then.

**Comment by christiangroth** [comment]
Would you mind just open the pull request so we can take a look on the changes and maybe discuss them?

**Comment by christiangroth** [comment]
@jckuester any suggestions on this?

**Comment by jckuester** [comment]
Would love to see a PR on supporting the GT-WT-01 variant  :) I can jump in and help if necessary.

@martinhoefling the homeduino project outputs results for all protocols that matches (https://github.com/pimatic/rfcontroljs/blob/master/src/controller.coffee#L143), but I like your approach of actually checking if the values make any sense instead of outputting results for multipe protocols. Good idea!

**Comment by martinhoefling** [comment]
@jckuester, @christiangroth  I currently only have a version with overwritten values. When looking through the code for possible discrimination, it was the first idea that came to my mind. I'll prepare something as soon as I find some time.

**Comment by christiangroth** [comment]
@martinhoefling Great, looking forward to it.

**Comment by martinhoefling** [comment]
#14 opened

**Comment by jckuester** [comment]
What do you think if make the weather-station read a config, eg. `./weather-station config.yml`:

```
sensors:
  1235:
    location: kitchen
    protocol: weather15
  3445:
    location: fridge
    protocol: weather12
```
If no config is provided, the weather-station would be in scanning mode and outputs all protocols that matches. One can use the scanning mode to find the correct protocol.

**Comment by martinhoefling** [comment]
Yes, that sounds reasonable. So you would get two lines, one with garbage and one with the correct values. This allows you to find the correct configuration.

**Comment by christiangroth** [comment]
Anything new on this one? Updated pull request or something?

**Comment by martinhoefling** [comment]
Sorry, I did not have time to implement this so far.

**Comment by martinhoefling** [comment]
closed by #16
